### PR TITLE
add: NodeBB

### PR DESCRIPTION
**Pull request by berksmbl**

### CODE REVIEW

**Review by Ponkhy — APPROVED** [review]
LGTM

File: templates/nodebb/assets/screenshot.png
Comment (Supernova3339) [inline templates/nodebb/assets/screenshot.png:1]: I would recommend you add additional screenshots.
Examples could include the front or backend.

Comment (Supernova3339) [reply templates/nodebb/assets/screenshot.png:1]: Let me know if you wish to implement this so I can approve it

Comment (berksmbl) [reply templates/nodebb/assets/screenshot.png:1]: The demo will suffice.

**Review by Supernova3339 — APPROVED** [review]
LGTM

**Comment by Tc-001** [comment]
Looks like it is not possible to add plugins with the default configuration, as they get overwritten once the server restarts.
Adding a volume mount pointing to `/usr/src/app/node_modules` seems to make it work, but I am unsure if that is the "correct" solution.

**Comment by berksmbl** [comment]
> Looks like it is not possible to add plugins with the default configuration, as they get overwritten once the server restarts. Adding a volume mount pointing to `/usr/src/app/node_modules` seems to make it work, but I am unsure if that is the "correct" solution.

I'm checking.

**Pull request merged**